app/lib/screens/SideMenuScreens/ToDo.dart

```diff
@@ -38,7 +37,16 @@ class _ToDoScreenState extends State<ToDoScreen> {
       floatingActionButton: FloatingActionButton(
         backgroundColor: ToDoTheme.floatingABColor,
         onPressed: () {
-          Navigator.pushNamed(context, '/addTodo');
+          ScaffoldMessenger.of(context).showSnackBar(
+            SnackBar(
```

Comment:
Since this will be frequently occuring widget, we can convert this into reusable method and can pass generic strings.

Reply:
yeah, you are right. I was just wondering if that's really worth is considering this is a pretty short piece of code. But once I started playing with the clear and adapting the UI, it kinda started to make sense. Will create this as an external widget.

Reply:
coming to think of it, this isn't really suitable as a widget, as it doesn't really render anything but would only show once-off message and I don't want it to wrap around other widgets to manage the `onPressed` side. I'll just convert this into a helper function.